subcommittees/README.md

```diff
@@ -0,0 +1,27 @@
+#Software Carpentry Foundation Subcommittees and Task Forces
+
+## Purpose and Rationale
+Software Carpentry is volunteer-powered organization that is rapidly growing. One of the greatest things about Software Carpentry is the huge diversity of people that have come together to collaborate, teach, and share ideas and resources. Your contributions to the community have made Software Carpentry a huge success!
+
+To meet the needs of our growing community, the 2015 Steering Committee Launched the first Subcommittee program to allow groups of members to organize and develop resources and activities that support our members. 
+
+The 2016 Steering Committee would like to encourage and invite the community to propose new initiatives in the form of subcommittees (a standing group for ongoing activities) and task forces (an ad hoc group focused on a finite task). Read on to learn more about existing initiatives and for information about how to propose a new initiatives that will shape our community.   
+
+## Current Subcommittees and Task Forces
+For information on existing and past subcommittees, please visit: https://github.com/swcarpentry/board/blob/master/subcommittees/subcommittees.md
```

Comment:
Could be more compact with `please visit [the subcommittee listing](subcommittees.md)`, which will also work when browsing forks, branches, earlier commits, etc. on GitHub.